apisix/plugins/request-id.lua

```diff
@@ -41,8 +41,7 @@ local schema = {
         header_name = {type = "string", default = "X-Request-Id"},
         include_in_response = {type = "boolean", default = true},
         algorithm = {type = "string", enum = {"uuid", "snowflake", "nanoid"}, default = "uuid"},
-        id_name = {type = "string", default = "req-id"},
-        get_from_query = {type = "boolean", default = false}
+        arg_name = {type = "string", default = "req-id"}
```

Comment:
We may set `header_name` and `arg_name` as exclusive, with the `oneOf` rule.

Reply:
sorry , I don't get it

Reply:
You can see https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L623-L631 as an example.

Reply:
I just want both `header_name` and `arg_name` to be configurable

https://github.com/apache/apisix/blob/b5a5d634b1c33fe652028f035df46bdd944573fb/apisix/plugins/request-id.lua#L229